packages/dicomImageLoader/src/shared/decodeImageFrame.ts

```diff
@@ -201,7 +201,11 @@ function postProcessDecodedPixels(
     Float32Array,
   };
 
-  if (options.targetBuffer && options.targetBuffer.type) {
+  const type = options.targetBuffer?.type;
+  const invalidTargetType =
+    isColorImage(imageFrame.photometricInterpretation) && type !== 'Uint8Array';
```

Comment:
why this is the case that we only care about uint8?

Reply:
DICOM only defines 8 bit representations of colour data, and all of the decoded versions produce 8 bit data.  If you can tell me how to figure out the right representation before calling the decode, then I can change this, but the call in StackViewport doesn't have any data to base the required representation on.

Reply:
can you please add that comment then in the code?

Reply:
Agreed to just check length and offset here and go to the next if otherwise.  This will result in not using the type here.